src/Serialization/main.jl

```diff
@@ -148,13 +148,37 @@ end
 ################################################################################
 # Version info
 
-function get_version_info()
-  result = Dict{Symbol, Any}(
-    :Oscar => ["https://github.com/oscar-system/Oscar.jl", VERSION_NUMBER]
-  )
+function file_version_info(dict::Dict{Symbol, Any})
+  ns = dict[:_ns]
+  version_info = ns[:Oscar][2]
+  return version_number(version_info)
+end
+
+function version_number(v_number::String)
+  return VersionNumber(v_number)
+end
+
+# needed for older versions 
+function version_number(dict::Dict)
+  return VersionNumber(dict[:major], dict[:minor], dict[:patch])
+end
+
+function file_version_info()
```

Comment:
To me this is an unintuitive name. I would expect it to return the version of some file, but it doesn't.

Maybe it can be called something like `current_version_info()` or `serialization_current_version()` or so?